ai/src/main/java/org/triplea/ai/doesnothing/DoesNothingAi.java

```diff
@@ -1,4 +1,4 @@
-package games.strategy.triplea.ai.weak;
+package org.triplea.ai.doesnothing;
```

Comment:
nit, does nothing is two words, canonical convention for the project is to word-split packages with periods.

IE: `org.triplea.ai.does.nothing`

I do like on the other hand that 'doesnothing' represents a singular entity, I think it's then a toss up. In case of a toss-up, the existing word-splitting practice I think wins the day.